digid_eherkenning/oidc/models/eherkenning.py

```diff
@@ -13,25 +12,60 @@
 )
 
 
-class EHerkenningConfig(OpenIDConnectBaseConfig):
+class AuthorizeeMixin(models.Model):
+    identifier_type_claim = ClaimField(
+        verbose_name=_("identifier type claim"),
+        # XXX: Anoigo specific default
+        default=ClaimFieldDefault("namequalifier"),
+        help_text=_(
+            "Claim that specifies how the legal subject claim must be interpreted. "
+            "The expected claim value is one of: "
+            "'urn:etoegang:1.9:EntityConcernedID:KvKnr' or "
+            "'urn:etoegang:1.9:EntityConcernedID:RSIN'."
+        ),
+    )
+    # TODO: what if the claims for kvk/RSIN are different claims names?
+    legal_subject_claim = ClaimField(
+        verbose_name=_("company identifier claim"),
```

Comment:
Not sure if the mismatch between the field name and `verbose_name`/`help_text` is intentional?

Reply:
intentional indeed - "legal subject" is meaningless to people configuring stuff in the admin, but in the code I want to use the terminology from the authentication context data model :) this claim will hold either a KVK nummer or RSIN (possibly other identifiers in the future, the eherkenning standard has more options than those two)